src/main/java/frc/robot/DistanceRange.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot;
+
+import java.util.HashMap;
+
+import frc.robot.AutonomousPath.VisionPathName;
+
+public class DistanceRange 
+  implements DistanceRangeInterface{
```

Comment:
This is overkill. We dont need both an interface and the implementation: for simple classes, just the class is ok. There's no seam needed, because we can just create one of these by hand